File: Exercises_Python/1983.py


numero = int(input())

maior = 0
matricula_vencedor = 0
for c in range(numero):
    matricula,nota = map(float, input().split())
    if nota > maior:
        maior = nota
        matricula_vencedor = matricula

if maior >= 8:
    print('{:.0f}'.format(matricula_vencedor))
else:
    print('Minimum note not reached')

    

File: Exercises_C++/1983.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

	int qntd,mat;
	double maior;
	cin >> qntd;
	maior = 0;
	mat = 0;

	for (int c=0 ; c < qntd ; c++) {

		long long matricula;
		double nota;

		cin >> matricula >> nota;
		if (nota > maior) {
			maior = nota;
			mat = matricula;
		}	
		
	}

	if (maior >= 8) {
		cout << mat << endl;
	}
	else {
		cout << "Minimum note not reached" << endl;
	}

}
